### ISSUE TITLE
Disabling the Restore Point Step

### ISSUE THREAD

**Issue by Blackyvk** [DriverPackSolution/DriverPack-Solution#74]
Is there a way to disable this  step ? it takes too long on some computers, so it could be nice to skip it ang directly start the installation of drivers and programs.

**Comment by skanskan** [comment]
There is an option but it doesn't work, it's just ignored.

**Comment by daedroza** [comment]
Agreed, making and compiling all such massive drivers database, but at the end, useless because it get's stuck.
